gltflib/gltf.py

```diff
@@ -91,12 +89,11 @@ def load_glb(cls: 'GLTF', filename: str, load_file_resources=False, resources: L
         with open(filename, 'rb') as f:
             return cls.read_glb(f, load_file_resources=load_file_resources, resources=resources,
                     encoding=encoding, basepath=basepath)
-        return gltf
```

Comment:
this code seems unreachable, due to the `return` on previous line. Same with the other `return gltf` below